templates/docs/content-guidelines.md

```diff
@@ -0,0 +1,190 @@
+
+To save space, you can use all numerals separated by a forward slash using the UK convention of [day]/[month]/[year].
+
+Use numerals for relative dates. For example, ‘2 hours ago’ or ‘Updated 5 days ago’.
+
+## Numbers
+
+Use numerals in the UI when it can help highlight an action and compare two amounts. Use the UK convention for commas and decimals in numbers. For example, £10.00 rather than £10,00.
+
+- **Example from Landscape:**
+
+<div class="u-fixed-width"><img src="https://assets.ubuntu.com/v1/135209d1-numbers-numberals-example-landscape.png" alt="An example of a dialaog in Landscape where the use of numbers is shown"></div>
+
+- Note: this guidance differs from the Copy Style Guide to spell out numerals less than 10. Follow the Copy Style Guide for copy that is not part of the UI.
+
+## Videos
+
+When you embed a video in a web page, you don’t need a ‘watch’ CTA as well. Use a clear header to explain to the user what they will get from clicking on the video. For example, what they will learn.
+
+## Writing notifications
+
+There are three basic concepts for UX copy that we use: attributes and objects, and actions.
+
+- **Attributes:** The components that make up a piece of content. For example, a title, description.
+- **Objects:** Usually nouns, the things users interact with.
+- **Actions:** Often verbs, things users can do with objects.
+
+<div class="u-fixed-width"><img src="https://assets.ubuntu.com/v1/cde94563-lxd-context.png" alt="A capture of a form of the LXD UI, including the app side navigation. The context relevant for an error message is outlined on the screenshot."></div>
+
+We should be specific about objects and actions and not have duplicate terms. Our messages typically include three attributes:
+
+- **Title**
+- **Description**
+- **CTA**
+
+**Example from Anbox:**
+
+<div class="u-fixed-width"><img src="https://assets.ubuntu.com/v1/e53e5133-reset-dialog.png" alt="Example of a reset dialog in Anbox with contextualized title, description and CTA."></div>
+
+You may need to consult with your product manager or engineers to clarify the objects and actions in different circumstances. Follow the guidance below to write messages that support users.
+
+### Title
+
+- Use a verb phrase to describe the action that will affect a certain object. In the example above, the action is to ‘reset’ and the object is ‘default values’.
+- Do not use punctuation in the title.
+
+### Description
+
+- Add details about the action, try not to add more objects.
+- Use punctuation in the description.
+
+**Research says to:**
+
+- Aim for 3-6 words per line, no more than 3 lines per message.
+- Avoid asterisks (users don’t really trust them).
+
+### CTAs
+
+- Match the text of the primary CTA button to the action in the title.
+- Note: our current convention is to make ‘Cancel’ the secondary CTA button. It performs the same action as an ‘X’ icon in the top right corner.
+
+## Notification library
```

Comment:
These may look better as an unordered list with linebreaks between the bold "item title" and the paragraph beneath it. Though I'm not sure if design has something else in mind, was there a mockup created for this page? CC @lyubomir-popov

Reply:
Lists can easily get messy and require a manual fine-tuning in HTML.

I think it's OK to leave it as is (with subheaders) for quick release, and we can finetune it later.